js/poloniex.js

```diff
@@ -465,17 +454,17 @@ module.exports = class poloniex extends Exchange {
             'ask': this.safeNumber (ticker, 'lowestAsk'),
             'askVolume': undefined,
             'vwap': undefined,
-            'open': open,
+            'open': undefined,
             'close': last,
             'last': last,
             'previousClose': undefined,
-            'change': change,
+            'change': undefined,
             'percentage': relativeChange * 100,
```

Comment:
We cannot multiply it by 100 if it is not defined. And since it can be `undefined` potentially, we have to check for that before multiplying.